feature/qos/otg_tests/egress_traffic_classification_and_rewrite_test/README.md

```diff
@@ -0,0 +1,205 @@
+
+@axelrod-mike	Reply...
+
+*   Configuration:
+    *   Configure Static MPLS LSP with MPLS pop and IPv4/IPv6 forward actions for a specific labels range (100020-100100).
+    *   Configure decapsulation rules for IPv4-DST-DECAP/32
+*   Traffic:
+    *   Generate IPoMPLSoGUE traffic from ATE Port 1 with labels between 100020 and 1000100
+*   Verfication:
+    *   Monitor telemetry on the DUT to verify that packets re being scehduled for transmission into correct forwarding groups.
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.4 Egress Classification and rewrite of IPv6oMPLSoGUE traffic with pop action
+*   Configuration:
+    *   Configure Static MPLS LSP with MPLS pop and IPv6 forward actions for a specific labels range (100020-100100).
+    *    Configure decapsulation rules for IPv6-DST-DECAP/12 
+*   Traffic:
+    *   Generate IPv6oMPLSoGUE traffic from ATE Port 1 with labels between 100020 and 1000100
+*   Verfication:
+    *   Monitor telemetry on the DUT to verify that packets re being scehduled for transmission into correct forwarding groups.
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.5 Egress Classification and rewrite of IPoMPLSoGRE traffic with pop action
+*   Configuration:
+    *   Configure Static MPLS LSP with MPLS pop and IPv4 forward actions for a specific labels range (100020-100100).
+    *   Configure decapsulation rules for IPv4-DST-DECAP/32
+*   Traffic:
+    *   Generate IPoMPLSoGRE traffic from ATE Port 1 with labels between 100020 and 1000100
+*   Verfication:
+    *   Monitor telemetry on the DUT to verify that packets re being scehduled for transmission into correct forwarding groups.
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.7 Egress Classification and rewrite of IPv6oMPLSoGRE traffic with pop action
+*   Configuration:
+    *   Configure Static MPLS LSP with MPLS pop and IPv6 forward actions for a specific labels range (100020-100100).
+    *    Configure decapsulation rules for IPv6-DST-DECAP/12 
+*   Traffic:
+    *   Generate IPv6oMPLSoGRE traffic from ATE Port 1 with labels between 100020 and 1000100
+*   Verfication:
+    *   Monitor telemetry on the DUT to verify that packets re being scehduled for transmission into correct forwarding groups.
+    *   Capture packets on the ATE's ingress interface to verify packet marking according to the marking table.
+    *   Analyze traffic flows to confirm that no packets are dropped on the DUT.
+
+### DP-1.17.8 Egress Classification and rewrite of IPoGRE traffic with pop action
```

Comment:
By "pop action" here do you mean GRE decapsulation? Can we use the latter terminology since I think it's more common?